guardian/commercial-templates
# remove legacy fabric-video template for all platforms

## What does this change?
As part of the migration process of templates to Svelte, we have successfully migrated the fabric-video template, as confirmed in the [migrate fabric-video PR](https://github.com/guardian/commercial-templates/pull/410). This PR removes the legacy fabric-video template.



## How to test
Run `pnpm build` followed by `pnpm preview` to verify that the fabric-video template link is no longer available in the Legacy section.


## How can we measure success?
Success can be measured by ensuring that all legacy code related to the fabric-video template is removed without any adverse effects on current templates. Additionally, the fabric-video template link should no longer be visible in the Legacy section of the test sandbox.


## Images
| Before |  After (fabric-video removed |
|-------|--------|
| <img width="831" alt="Screenshot 2024-12-12 at 11 40 58" src="https://github.com/user-attachments/assets/ed63e1be-88c0-4c7e-a2db-e996d50b4cb4" />| <img width="851" alt="Screenshot 2024-12-12 at 11 44 21" src="https://github.com/user-attachments/assets/c15e890a-6faa-41a8-bbd1-b8d173e2f28e" /> |



## Accessibility
